assets/scss/base/forms.scss

```diff
@@ -0,0 +1,8 @@
+.form-control {
+  background: rgba(0, 0, 0, .1);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.1)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.1` should be written with a leading zero as `0.1`